src/urdf_loader.h

```diff
@@ -24,16 +27,19 @@ namespace or_urdf
   {
   public:
     /** Opens a URDF file and returns a robot in OpenRAVE */
-    bool load(std::ostream& sout, std::istream& sin);
+    bool loadFile(std::ostream& sout, std::istream& sin);
+    bool loadJsonString(std::ostream& sout, std::istream& sin);
     
     /** Constructs plugin and registers functions */
     URDFLoader(OpenRAVE::EnvironmentBasePtr env) : OpenRAVE::ModuleBase(env)
     {
       __description = "URDFLoader: Loader that imports URDF files.";
       _env = env;
 
-      RegisterCommand("load", boost::bind(&URDFLoader::load, this, _1, _2),
+      RegisterCommand("load", boost::bind(&URDFLoader::loadFile, this, _1, _2),
```

Comment:
This looks great, but I think the names are now confusing. I proposed that we (a) rename `URDFLoader::load` -> `URDFLoader::loadModel`, (b) rename `URDFLoader::loadFile` -> `URDFLoader::loadURI` and rename its command registration to `LoadURI`, and (c) create a new wrapper command called `URDFLoader::load` registered as `Load` which internally forwards to `LoadURI` (perhaps eventually with a deprecation warning?). What do y'all think?